app/controllers/users_controller.rb

```diff
@@ -28,4 +51,25 @@ def user_params
       :password_confirmation)
   end
 
+  def load_user
+    @user = User.find_by id: params[:id]
+    unless @user
```

Comment:
```suggestion
    return if @user
```
Các dòng dưới viết luôn khỏi có block nhìn rườm rà